code/modules/cm_marines/orbital_cannon.dm

```diff
@@ -384,12 +384,10 @@ var/obj/structure/ship_rail_gun/almayer_rail_gun
 	warhead_kind = "incendiary"
 	icon_state = "ob_warhead_2"
 
+
 /obj/structure/ob_ammo/warhead/incendiary/warhead_impact(turf/target, inaccuracy_amt = 0)
-	var/range_num = max(8 - inaccuracy_amt*2, 3)
-	for(var/turf/TU in range(range_num,target))
-		for(var/obj/flamer_fire/F in TU) // No stacking flames!
-			qdel(F)
-		new/obj/flamer_fire(TU, 10, 50) //super hot flames
+	var/range_num = max(20 - inaccuracy_amt * 2, 15)
+	flame_radius(range_num, target, 10, 20, 10, 10)
```

Comment:
Unless I'm mistaken here, these values would make the Incendiary OB weaker and shorter lasting than it actually is. But again, flame_radius is clustered with gaussians, so I might have read it wrong.

Reply:
Tested the burn values in-game for potency, the default was an instant kill basically.